### ISSUE TITLE
Copy Kubectl to /usr/local/bin hanging

### ISSUE THREAD

**Issue by iamadrigal** [NVIDIA/deepops#1204]
Hi there, I tried to deploy k8s today using a dgx station as mgmt01 (just for the CPU) and then gpu01 on a server with 8 GPUs.

ansible-playbook -l k8s-cluster playbooks/k8s-cluster.yml --skip-tags nvidia

I ran the playbook while using a different control machine. It made it all the way to this section below, then hung forever on this issue:

`PLAY [k8s-cluster] *************************************************************************************************************************

TASK [check for kubectl] *******************************************************************************************************************
ok: [mgmt01 -> localhost]

TASK [modify kubectl permissions] **********************************************************************************************************
ok: [mgmt01 -> localhost]

TASK [copy kubectl to /usr/local/bin] ******************************************************************************************************
Sorry, try again.
`

I had to run reset.yml to try again, and it didn't work again. I did check that both machines had rsync.
The issue was resolved by running the playbook on mgmt01 (the dgx station) but wanted to know what might cause this issue since a future use case might be using a separate machine as control.

**Comment by dholt** [comment]
I don't think I've ever seen it hang here, it usually works or doesn't work, and will fall back to some other things if it fails. You can try running your Ansible command with `-vvv` to see if it will tell you which command is hanging.

**Comment by TE-0000144078** [comment]
I met the same problem. You can solve to use "sudo ansible-playbook,,,"

**Comment by sugupoko** [comment]
I met the same problem. You can solve to use "sudo ansible-playbook,,,"

**Comment by Suthek** [comment]
>  The issue was resolved by running the playbook on mgmt01 (the dgx station) but wanted to know what might cause this issue since a future use case might be using a separate machine as control.

I have had the very same issue a short while back; it seems the problem was that the host machine (where the script was running on) had a different sudo/become password than the servers I was installing deepops on. Once I changed my local account's password to be the same as the ansible accounts on the servers, this step ran through smoothly.

I cannot guarantee that this was in fact the cause of the issue, but that was the only action I took between it not working and it working.
